src/magic.erl

```diff
@@ -1,3 +1,4 @@
+%% @doc this comment is really really really too long, you should reject this PR, @elvisci. Please, oh please. Reject it, I tell you.
```

Comment:
Line 1 is too long: %% @doc this comment is really really really too long, you should reject this PR, @elvisci. Please, oh please. Reject it, I tell you..